src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.ServiceModel.Duplex.csproj">
-      <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net45;win8;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Only update these if you are *retargeting* your reference assemblies.  These are supposed to represent the minimum frameworks you expect to be supported by the API version of that reference.  See https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/package-projects.md#applicability-validation